.github/workflows/openscap_multi_arch.yml

```diff
@@ -0,0 +1,76 @@
+name: Multiarch build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+env:
+  IMAGE_NAME: openscap-ocp
+  IMAGE_TAG: multi-arch
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_NAMESPACE: compliance-operator
```

Comment:
I don't think this is the right namespace. The other jobs seem to specify this differently. https://github.com/ComplianceAsCode/compliance-operator/blob/master/.github/workflows/release.yml#L13

Reply:
@mrogers950  i have updated namespace to "complianceascode" as per given URL. can you please review it once.